src/components/dashboard/DepositTokens/DepositTokensForm.jsx

```diff
@@ -62,7 +63,7 @@ function mapStateToProps (state) {
   const tokens = state.get(DUCK_TOKENS)
   const { selectedNetworkId, selectedProviderId } = state.get(DUCK_NETWORK)
 
-  const token = tokens.item(tokenId)
+  const token = tokens.item(tokenId || TIME)
```

Comment:
Is it okay? What tokens can we get there except TIME ?

Reply:
it is okay for now